package/gluon-core/luasrc/usr/lib/lua/gluon/wireless.lua

```diff
@@ -61,10 +61,11 @@ function M.foreach_radio(uci, f)
 	for index, radio in ipairs(radios) do
 		local band = radio.band
 
+		-- radio index is zero-based
```

Comment:
A more descriptive commit message would be great - you are not "correcting" the index, you are changing the interpretation of the index from 1-based to 0-based.

Reply:
Done - if there is nothing else, this is ready to merge.